docs/user-documentation/glossary.md

```diff
@@ -1,95 +1,202 @@
 
 ## Open Source
-Open source describes a method of software development that promotes access to the end product's source code. Islandora is an open source product with an active development community, operating under the GPL license (2.0) for Drupal components and the MIT license for non-Drupal components. 
- 
+Open source describes a method of software development that promotes access to the end product's source code. Islandora is an open source product with an active development community, operating under the [GPL license (2.0)](https://www.gnu.org/licenses/old-licenses/gpl-2.0.html) for [Drupal](#drupal) components and the [MIT license](https://opensource.org/licenses/MIT) for non-[Drupal](#drupal) components. 
+
+## PR
+See [Pull request](#pull-request)
+
+## Pull request
+Also PR; sometimes also known as merge requests; technical term from distributed version control systems for software code like Git. Code contributors can request that the maintainer of a code repository 'pulls' the code change into the repository after approval.
+
+## RDF
+See [Resource Description Framework](#resource-description-framework)
+
+## Repository Item
+See: [Resource Node](#resource-node)
+
+## Resource Description Framework
+Also RDF; family of World Wide Web Consortium (W3C) specifications originally designed as a data model for metadata. It has come to be used as a general method for conceptual description or modeling of information that is implemented in web resources. The data is modelled as a set of statements, also known as triples. A collection of RDF statements intrinsically represents a directed graph. Data represented according to the RDF specifications can be serialized in different ways, for instance using [JSON-LD](#json-ld).
+
 ## Resource Node
-A resource node is a Drupal node that represents a single conceptual item or object stored in an Islandora repository. It acts as a container for all files and metadata associated with that item, and is the place where the item 'lives' as a visitable url. 
+A resource node is a [Drupal](#drupal) [Node](#node) that represents a single conceptual item or object stored in an Islandora repository. It acts as a container for all files and metadata associated with that item, and is the place where the item 'lives' as a visitable URI. 
```

Comment:
Maybe clarify that this is an Islandora-specific concept? If it indeed is! (That is my current understanding, anyway). 

Link to https://islandora.github.io/documentation/user-documentation/resource-nodes/ ?

Maybe one important thing missing here is that Resource Nodes get important system behaviors attached to them, when they are created or updated. These behaviors would include derivative generation, fixity check, indexing/reindexing, sync to Fedora, etc.

Reply:
That is my understanding as well, this is Islandora-specific terminology. I added two sentences to the entry to make this clearer: 

> The term 'Resource node' is specific to Islandora. Typically, Resource nodes in an Islandora installation will use a specific [Content type](#content-type) for the digital assets stored in the repository.